### PR TITLE
feat(core): Added sd-keyboard-modal for displaying active keyboard sh…

### PR DESCRIPTION
**Pull request by eozan**
…ortcuts, changed keyboardManager accordingly.

SD-2667

### CODE REVIEW

File: client/app/scripts/superdesk/keyboard/views/keyboard-modal.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div sd-modal data-model="enabled" class="add-scheme-modal modal-strech modal-big">
+    <div class="modal-header">
+        <h3 translate>Keyboard Shortcuts</h3>
+    </div>
+    <div class="modal-body">
+        <section ng-repeat="(group, items) in data">
+            <h4>{{group}}</h4>
+            <ul>
+                <li ng-repeat="item in items" ng-hide="item.opt.hide">
+                    <span class=""><b>{{item.label}}</b></span>
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk/keyboard/views/keyboard-modal.html]: One time binding?

File: client/app/scripts/superdesk/editor/editor.js [hunk 2]
```diff
@@ -600,7 +601,8 @@ angular.module('superdesk.editor', ['superdesk.editor.spellcheck'])
                     scope.medium = new window.MediumEditor(scope.node, editorOptions);
 
                     scope.$on('spellcheck:run', render);
-                    scope.$on('key:ctrl:shift:d', render);
+                    //scope.$on('key:ctrl:shift:d', render);
```
Comment (mdhaman) [inline client/app/scripts/superdesk/editor/editor.js]: pls. can you remove the comment.

File: client/app/scripts/superdesk-workspace/content/content.js [hunk 3]
```diff
@@ -126,12 +129,20 @@
                     });
                 });
 
+                /*
```
Comment (mdhaman) [inline client/app/scripts/superdesk-workspace/content/content.js]: pls. can you remove the comment.

**Comment by mdhaman** [comment]
:+1: except minor comments

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**